integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -279,16 +276,23 @@ public int getattr(String path, FileStat stat) {
       stat.st_mtim.tv_sec.set(ctime_sec);
       stat.st_mtim.tv_nsec.set(ctime_nsec);
 
-      // for shell-based group mapping, use the uid and gid of the corresponding user registered in
-      // unix; otherwise use uid and gid of the user running alluxio-fuse
+      // for shell-based group mapping, try to get the uid and gid of file/dir owner.
+      // If the shell-based group mapping is not enabled in Fuse
+      // or the user does not registered in the unix,
+      // uid and gid of the user running alluxio-fuse will be set.
+      long uid = -1;
```

Comment:
could simplify the code a little by setting these to the default `UID`/`GID` values

Reply:
The user group translation rule seems to affect `getAttr` most, so I reconstruct the docs here. PTAL.